persistence-jdbc/scaladsl/src/test/scala/com/lightbend/lagom/scaladsl/persistence/jdbc/JdbcPersistenceSpec.scala

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package com.lightbend.lagom.scaladsl.persistence.jdbc
+
+import akka.actor.ActorSystem
+import akka.cluster.Cluster
+import com.lightbend.lagom.internal.persistence.jdbc._
+import com.lightbend.lagom.internal.scaladsl.persistence.ReadSideConfig
+import com.lightbend.lagom.persistence.{ ActorSystemSpec, PersistenceSpec }
+import com.lightbend.lagom.scaladsl.persistence.jdbc.testkit.TestUtil
+import com.typesafe.config.{ Config, ConfigFactory }
+import play.api.db.Databases
+import play.api.{ Configuration, Environment }
+
+import scala.concurrent.Await
+import scala.concurrent.duration._
+import scala.util.Random
+
+abstract class JdbcPersistenceSpec(_system: ActorSystem) extends ActorSystemSpec(_system) {
```

Comment:
It uses `JdbcSessionImpl`, `JdbcReadSideImpl`, `TestUtil` and `ReadSideConfig` which are dsl-specific and is only ~40 lines of code, so not super easy/rewarding to factor out common behaviour.